artemis/reporting/modules/nuclei/translations/nuclei_messages/pl_PL.py

```diff
@@ -534,6 +534,13 @@
     "[no description] http/vulnerabilities/jenkins/unauthenticated-jenkins.yaml": "Wykryto publicznie dostępny system Jenkins, który nie wymaga zalogowania. Rekomendujemy, aby nie był on dostępny publicznie oraz zalecamy włączenie mechanizmu uwierzytelniania, ponieważ może umożliwić atakującemu uruchamianie własnych zadań lub nieuprawniony dostęp do informacji/kodu źródłowego.",
     "Possible Juicy Files can be discovered at this endpoint. Search / Grep for secrets like hashed passwords ( SHA ) , internal email disclosure etc.": "Wykryto konfigurację Adobe Experience Manager, mogącą zawierać dane, które nie powinny być dostępne publicznie.",
     "[no description] http/takeovers/webflow-takeover.yaml": "Wykryto domenę kierującą do narzędzia Webflow, ale domena docelowa jest wolna. Atakujący może zarejestrować domenę w narzędziu Webflow, aby serwować tam swoje treści. Jeśli domena nie jest używana, rekomendujemy jej usunięcie.",
+    "HiBoss allows remote unauthenticated attackers to cause the server to execute arbitrary code via the 'server_ping.php' endpoint and the 'ip' parameter.": "Wykryto, że narzędzie HiBoss umożliwia niezalogowanym atakującym zdalne wykonanie kodu."
```

Comment:
```suggestion
    "HiBoss allows remote unauthenticated attackers to cause the server to execute arbitrary code via the 'server_ping.php' endpoint and the 'ip' parameter.": "Wykryto narzędzie HiBoss, które umożliwia niezalogowanym atakującym zdalne wykonanie kodu."
```

Reply:
piszę w tej formie celowo, bo nie wiem czy to jest cecha narzędzia HiBoss, czy niektó©ych z jego wersji/konfiguracji  a w templatce to nie jest napisane